fspiop-api/documents/API-Definition_v1.1.1.md

```diff
@@ -2415,9 +2415,7 @@ Alternative URI: N/A
 
 Logical API service: **Notify Transfer Information Error**
 
-If the server is unable to commit a transfer, or another processing or validation error occurs, during the processing of a **PUT /transfers/**_{ID}_ callbac, the notification **PATCH**
-
-**/transfers/**_{ID}_**/error** is used. The _{ID}_ in the URI should contain the **transferId** (see [Table 30](#table-30)) that was used for the creation of the transfer, or the _{ID}_ that was used in the [**GET /transfers/**_{ID}_](#6731-get-transfersid). See [Table 34](#table-34) for data model.
+If the server is unable to commit a transfer, or another processing or validation error occurs, during the processing of a **PUT /transfers/**_{ID}_ callback with transferState as COMMITTED, the notification **PATCH** **/transfers/**_{ID}_**/error** is used. The _{ID}_ in the URI should contain the **transferId** (see [Table 30](#table-30)) that was used for the creation of the transfer. See [Table 34](#table-34) for data model.
```

Comment:
> ... , the notification **PATCH** **/transfers/**_{ID}_**/error** is used.

Can we change the "is used" to "can be used (depending on scheme rules)" (or similar) so that this is not a requirement in the API, rather an optional feature that can be used based on scheme rules? This makes the change request more compatible with version 1.0, where no such error notifications are sent.